src/react/src/features/router/Router.tsx

```diff
@@ -23,23 +23,25 @@ export function Router() {
         dispatch(handleSelect(val));
     };
     //@todo need to add a tab for account
+    //@todo redirect back to login if a user isn't logged in
     return (
         <BrowserRouter basename='/'>
-            <Tabs value={tabValue} onChange={handleTabs} centered>
-                {tabComponent}
-            </Tabs>
             <Switch>
-                <Route exact path="/">
-                    <Redirect to="/collection" />
+                <Route exact path='/' component={Login} />
```

Comment:
path should be /login

Reply:
we can add a redirect to `/login`. but we need to show login at `/` too or else nothing will show up

Reply:
That's what the redirect is for